### PR TITLE
Fixed: About,Devices,Support,Blog page doesn't renders from footer through Team Page 

### LINKED ISSUE #3557: About,Devices,Support,Blog page doesn't renders from footer through Team Page
**Actual Behaviour**

When a user goes on the team page and afterwards tap on the About, Devices, Blog or Support button from the footer segment, then it does not renders the respective pages, but on reloading it does.

**Expected Behaviour**

The About, Devices, Blog and Support page must be rendered on clicking their respective button from the footer segment on accessing through the Team page 

**Steps to reproduce it**

Go to the Team page.
Now, click on the About, Devices, Blog or Support button from the footer section.

**Would you like to work on the issue?**

Yes, I would like to work on the issue.

### PR DESCRIPTION
**Pull request by 927tanmay**
Fixes #3557  

Changes: Changed the Link tags in the footer with the hyperlink tags.

Demo Link: https://pr-3557-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/About/Team/index.js [hunk 1]
```diff
@@ -133,26 +133,26 @@ const Description = styled.div`
...
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
 
-    let data = await getContributors();
-    let flags = [],
-      output = [];
-    for (let i = 0; i < data.length; i++) {
-      if (flags[data[i].name]) {
-        continue;
+    async function fetchData() {
```
Comment (anishagg17) [inline src/components/About/Team/index.js:141]: here `IIFE` can be used

Comment (927tanmay) [reply src/components/About/Team/index.js:141]: Using this also have the same effect as of IIFE, for using async function in hooks this is the recommended way. We can use either of them. Now it solves the problem of without having the need to reload the page.

File: surge_deploy.sh [hunk 2]
```diff
@@ -17,3 +17,12 @@ export SURGE_TOKEN=a34fd7119aacd204150edd6a8e29903d
 
 export DEPLOY_DOMAIN=https://pr-${TRAVIS_PULL_REQUEST}-fossasia-susi-web-chat.surge.sh
 surge --project ./build --domain $DEPLOY_DOMAIN;
+
```
Comment (akshatnitd) [inline surge_deploy.sh:20]: Remove these extra blank lines. Rest looks good!

Comment (927tanmay) [reply surge_deploy.sh]: @akshatnitd please check again I have removed the extra spaces

File: src/components/About/Team/index.js [hunk 3]
```diff
@@ -133,26 +133,25 @@ const Description = styled.div`
...
     document.title =
       'Developer Team of SUSI.AI - Open Source Artificial Intelligence for Personal Assistants, Robots, Help Desks and Chatbots';
 
-    let data = await getContributors();
-    let flags = [],
-      output = [];
-    for (let i = 0; i < data.length; i++) {
-      if (flags[data[i].name]) {
-        continue;
+    (async function fetchData() {
```
Comment (anishagg17) [inline src/components/About/Team/index.js:141]: Why to name the function, `fetchData` is a very common function name and can be used for future.

Comment (927tanmay) [reply src/components/About/Team/index.js]: @anishagg17 please review I have changed the function the name.

**Pull request merged**